src/renderer/components/TimelineSpace/Contents/Cards/Toot.vue

```diff
@@ -1,5 +1,9 @@
 <template>
-  <div class="toot" tabIndex="0">
+<div class="status" tabIndex="0">
+  <div v-if="filterd(message)" class="filterd">
```

Comment:
__[Misspell]__

"filterd" is a misspelling of "filtered"


<!-- Comment by [Sider](https://sider.review) -->